packages/grafana-data/package.json

```diff
@@ -14,13 +14,15 @@
   "main": "src/index.ts",
   "types": "src/index.ts",
   "scripts": {
-    "tslint": "tslint -c tslint.json --project tsconfig.json",
+    "lint": "eslint src/ --ext=.js,.ts,.tsx",
     "typecheck": "tsc --noEmit",
     "clean": "rimraf ./dist ./compiled",
     "bundle": "rollup -c rollup.config.ts",
     "build": "grafana-toolkit package:build --scope=data"
   },
   "devDependencies": {
+    "@grafana/eslint-config": "^1.0.0-pre",
+    "@grafana/tsconfig": "^1.0.0-pre",
```

Comment:
Should this be: [@grafana/tsconfig-grafana](https://github.com/grafana/tsconfig-grafana/blob/master/package.json#L4)

Reply:
Yes, I haven't renamed/moved them yet.